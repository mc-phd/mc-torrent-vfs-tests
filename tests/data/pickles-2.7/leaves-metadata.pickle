(dp1
S'info'
p2
(dp3
S'length'
p4
I362017
sS'piece length'
p5
I16384
sS'name'
p6
S'Leaves of Grass by Walt Whitman.epub'
p7
sS'pieces'
p8
S'\x1f\x9c?Y\xbe\xec\x07\x97\x15\xecS2K\xde\x85i\xe4\xa0\xb4\xeb\xecB0}L\xe5U{]9d\xc5\xefU\xd3T\xcfJn\xcc{\xf1\xbc\xafy\xd1\x1f\xa5\xe0\xbe\x06Y<\x8f\xaa\xfc\x0c+\xa2\xcfv\xd7\x1c[\x01Rk#\x00\x7f\x9e\x99)\xbe\xaf\xc5\x15\x1ee\x11\t1\xa1\xb4L!\xbf\x1eh\xb9\x13\x8f\x90I^i\r\xbcU\xf5r\xe4\xc2\x94L\xba\xcf&\xe6\xb3\xae\x8ar)\xd8\x8a\xaf\xa0_a\xea\xaej\xbf?\x07\xcbm\xb9g|\xc6\xad\xedM\xd3\x98^E\x86\'V\x7f\xa7c\x9f\x06_q\xb1\x89T0J\xcacfr\x9e\x0bGs\xd7z\xe8\x0c\xaa\x96\xa5$\x80M\xfeK\x9b\xd3\xde\xae\xf9\x99\xc9\xddQ\x02tgQ\x9d^\xb2V\x1a\xe2\xcc\x01F}\xe5\xf6C\n`\xbc\xba$yv\x92\xef\xa8w\r#\xdf\n\x83\r\x91\xcb5\xb3@z\x88\xba\xa0Y\r\xc8\xc9\xaaj\x12\x0f\'Cg\xdc\xd8g\xe8\x8e\x838\xc5r\xa0n<\x80\x1b)\xf5\x19\xdfS+>v\xf6p\xcfj\xeeS\x10\x7f=97\x84\x83\xf6\x9c\xf8\x0f\xa5h\xb1\xea\xc5;PaY\xe9\x88\xd8\xbc\x16\x92-\x12]w\xd8\x03\xd6R\xc3\xca0p\xc1n\xed\x91r\xabPm \xe5"\xea?\x1a\xb6t\xb3\xf9#\xd7o\xe8\xf4O\xf3.7,;7ed\xc6\xfb_\r\xbeR\x16O\x03b\x9f\xd12&6\xba\xbb,\x01K}\xaeX-\xa4\x13c\x96Ra\xe6\xce\x12\xb47\x01\xf0\xa8\xc9\xed\x15 \xa7\x0e\xba\x00D\x00\xa2gv_m=\xd5\xc7\xbe\xb5\xbd<u\xf3\xdf*TV\na\x80\x11G\xfaN\xc7\xcfV\x8ep:\xcb\x04\xe5a\nMV\xdc\xc2B\xd02\x93\xe9Dl\xf5\xe4W\xd8\xeb=\x95\x88\xfd\x90\xc6\x98\xde\x9b\r\xad\x92\x98\t\x06\xc0&\xd8\xc1@\x8f\xa0\x8f\xe4\xec'
p9
ssS'infoHash'
p10
S'd2474e86c95b19b8bcfdb92bc12c9d44667cfa36'
p11
sS'announce-list'
p12
(lp13
s.