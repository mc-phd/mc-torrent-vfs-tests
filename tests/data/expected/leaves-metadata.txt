dr-xr-xr-x 1 user group 0 03-16-2016 19:33 .META
-r--r--r-- 1 user group 0 03-16-2016 19:33 .META/announce-list
-r--r--r-- 1 user group 5 03-16-2016 19:33 .META/piece length
-r--r--r-- 1 user group 362017 03-16-2016 19:33 Leaves of Grass by Walt Whitman.epub
